tests/runner-tests/expr/assign/struct_align.c

```diff
@@ -0,0 +1,5 @@
+// succeeds
+int main() {
+	struct S { int *y, z; } s, ss;
+	s = ss;
```

Comment:
Can you also check that it correctly copies?
```suggestion
// code: 1
int main() {
	struct S { int *y, z; } s, ss;
	ss.z = 1;
	s = ss;
	return s.z;
```